Formula/liboqs.rb

```diff
@@ -31,17 +31,23 @@ class Liboqs < Formula
   fails_with gcc: "5"
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args, "-GNinja", "-DBUILD_SHARED_LIBS=ON"
-      system "ninja"
-      system "ninja", "install"
-    end
-    pkgshare.install "tests"
+    # The build fails on Catalina when enabling this.
+    ENV.runtime_cpu_detection if MacOS.version >= :big_sur || OS.linux?
+    system "cmake", "-S", ".", "-B", "build",
+                    "-G", "Ninja",
+                    "-DBUILD_SHARED_LIBS=ON",
+                    "-DOQS_USE_OPENSSL=ON",
+                    "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}",
+                    "-DOQS_DIST_BUILD=ON", # enable runtime cpu detection
```

Comment:
Doesn't work for some reason. Strange.

Reply:
Oh, wait, it does. Except it breaks the audit, since it just scans for `ENV.runtime_cpu_detection` without actually checking whether it's applied on that platform. That's actually why I wanted to set `OQS_DIST_BUILD` to `ON` even for Catalina.